ethpm/backends/registry.py

```diff
@@ -50,10 +50,10 @@ class RegistryURIBackend(BaseURIBackend):
     def __init__(self) -> None:
         from web3 import (
             Web3,
-            WebsocketProvider,
+            WebsocketProviderLegacy,
```

Comment:
Good thought. `ethpm` is deprecated though and being removed in `v7` so I don't think it will be a good time investment here. If you see this somewhere in the tests else that would be good to note.